ui/components/multichain/account-details/account-details-authenticate.js

```diff
@@ -0,0 +1,80 @@
+import React, { useCallback, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  DISPLAY,
+  SEVERITIES,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import {
+  BannerAlert,
+  ButtonPrimary,
+  ButtonSecondary,
+  Text,
+  TextField,
+} from '../../component-library';
+import Box from '../../ui/box/box';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { exportAccount, hideWarning } from '../../../store/actions';
+
+export const AccountDetailsAuthenticate = ({ address, onCancel }) => {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+
+  const [password, setPassword] = useState('');
+
+  // Password error would result from appState
+  const warning = useSelector((state) => state.appState.warning);
+
+  const onSubmit = useCallback(() => {
+    dispatch(exportAccount(password, address)).then((res) => {
+      dispatch(hideWarning());
+      return res;
+    });
+  }, [dispatch, password, address]);
+
+  return (
+    <>
+      <Text marginTop={6} variant={TextVariant.bodySm}>
+        {t('enterYourPassword')}
+      </Text>
+      <TextField
+        type="password"
+        onInput={(e) => setPassword(e.target.value)}
+        placeholder={t('password')}
+        onKeyDown={(e) => {
+          if (e.key === 'Enter') {
+            onSubmit();
+          }
+        }}
+        autoFocus
+      />
+      {warning ? (
+        <Text
+          marginTop={1}
+          color={TextColor.errorDefault}
+          variant={TextVariant.bodySm}
+        >
+          {warning}
+        </Text>
+      ) : null}
```

Comment:
Can we use` FormTextField` here rather than handling label, error with Text separately. 
```suggestion
  const handleKeyPress = (e) => {
    if (e.key === 'Enter') {
      onSubmit();
    }
  };
  
  return (
    <>
      <FormTextField
        marginTop={6}
        id="account-details-authenticate"
        label={t('enterYourPassword')}
        error={warning}
        helpText={warning && warning}
        onChange={(e) => setPassword(e.target.value)}
        value={password}
        inputProps={{
          onKeyPress: handleKeyPress,
        }}
      />
```

Reply:
Nice suggestion!